sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -46,5 +46,14 @@ internal TextAnalyticsRequestOptions(bool includeStatistics, string modelVersion
         /// <see cref="StringIndexType.Utf16CodeUnit"/>.
         /// </summary>
         public StringIndexType StringIndexType { get; set; } = StringIndexType.Utf16CodeUnit;
+
+        /// <summary>
+        /// If set to true (the default for `StartAnalyzeHealthcareEntities` and `RecognizePiiEntities`),
+        /// you opt-out of having your text input logged for troubleshooting. By default, Text Analytics
+        /// logs your input text for 48 hours, solely to allow for troubleshooting issues.
+        /// Setting this parameter to true, disables input logging and may limit our ability
+        /// to investigate issues that occur.
```

Comment:
"By default, TA logs your output", but it seems `StartAnalyzeHealthcareEntities` and `RecognizePiiEntities` disable logging by default. Is this something we do on the client side or are these two exceptions?

Reply:
Also, I think the back quote ` doesn't do anything special in docstrings. Maybe we should use single or double quotes:
```
the default for 'StartAnalyzeHealthcareEntities' and "RecognizePiiEntities"
```
or maybe even add a reference to the related methods with `cref` if applicable.

Reply:
all is done on the service side. those two are exceptions. I will phrase it differently and see if I can make it more clear

Reply:
updated